include/IECore/CubicBasis.h

```diff
@@ -60,6 +69,8 @@ class IECORE_EXPORT CubicBasis
 
 		CubicBasis( const MatrixType &m, unsigned s );
 
+		CubicBasis( StandardCubicBasis stockBasis );
```

Comment:
seems the variable name should be renamed as well

Reply:
Gah. Fixed in 0a75430